weasyl/define.py

```diff
@@ -337,8 +337,8 @@ def captcha_verify(form):
         response=form.g_recaptcha_response,
         remoteip=get_address())
     response = http_post('https://www.google.com/recaptcha/api/siteverify', data=data)
-    result = response.json()
-    return result['success'] == "true"
+    jsonResult = response.json()
+    return jsonResult['success'][0] == "true"
```

Comment:
Slight progress? But not there yet. 

https://gist.github.com/kfkitsune/ea2a1ab08449c7ecc7a4b936e6efaaed

2016-06-11T03:59:37+0000 [stderr#error]   File "/vagrant/weasyl/controllers/user.py", line 105, in POST
2016-06-11T03:59:37+0000 [stderr#error]     if not define.captcha_verify(form):
2016-06-11T03:59:37+0000 [stderr#error]   File "/vagrant/weasyl/define.py", line 341, in captcha_verify
2016-06-11T03:59:37+0000 [stderr#error]     return jsonResult['success'][0] == "true"
2016-06-11T03:59:37+0000 [stderr#error] TypeError: 'bool' object has no attribute '__getitem__'